src/ref.ts

```
@@ -36,6 +36,14 @@
 };
 
 export const supportRef = (nodeOrComponent: any): boolean => {
+  // React 19 no need `forwardRef` anymore. So just pass if is a React element.
+  if (
+    isReactElement(nodeOrComponent) &&
+    (nodeOrComponent as any).props.propertyIsEnumerable('ref')
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'propertyIsEnumerable' from target object.



(lint/suspicious/noPrototypeBuiltins)